### PR TITLE
Added Relay

### PR DESCRIPTION
**Pull request by gsasouza**

### CODE REVIEW

File: src/components/PersonList.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <FlatList
+      data={props.persons}
+      renderItem={({ item }) => (
+        <ListItem
+          title={item.name}
+          rightIcon={{name: 'chevron-right'}}
+          // onPressRightIcon={props.navigation.navigate(props.route, item)}
+        />
+      )}
+      keyExtractor={() => (Math.round(Math.random()*100))}
```
Comment (jgcmarins) [inline src/components/PersonList.js]: you can use `person.id`

File: src/screens/__generated__/PersonScreenQuery.graphql.js [hunk 2]
```diff
@@ -0,0 +1,223 @@
+/**
```
Comment (jgcmarins) [inline src/screens/__generated__/PersonScreenQuery.graphql.js:1]: add `__generated__` to `.gitignore`

**Review by jgcmarins — CHANGES_REQUESTED** [review]
almost there

File: src/components/PersonList.js (hunk 1 above)
Comment (gsasouza) [reply src/components/PersonList.js]: the problem is that person doesn't have an id. Can I use this ```keyExtractor={ (item, index) => index } ```` ?

File: src/relayNetwork.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  return await response.json();
+}
+
+const source = new RecordSource();
+const store = new Store(source);
+const network = Network.create(fetchQuery);
+const handlerProvider = null;
+
+const environment = new Environment({
+  handlerProvider,
```
Comment (jgcmarins) [inline src/relayNetwork.js:30]: try to remove this

**Review by jgcmarins — APPROVED** [review]
looks good!
great job 💯

File: src/components/PersonList.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { List, ListItem } from 'react-native-elements';
+import { FlatList } from 'react-native';
+
+export default (props) => (
+  <List>
+    <FlatList
```
Comment (matheuscorreia) [inline src/components/PersonList.js]: A list inside a list? 
Listception.
Try to refactor this to use only react-native-elements like [this](https://react-native-training.github.io/react-native-elements/API/lists/#using-map-function-implemented-with-link-and-icon), or use only FlatList.

**Review by matheuscorreia — APPROVED** [review]
Awesome! :tada:

File: src/components/PersonList.js (hunk 4 above)
Comment (jgcmarins) [reply src/components/PersonList.js]: Good point. It's my bad. He copied from a poor example I've shared with him.

**Review by jgcmarins — APPROVED** [review]
Looks good

**Review by jgcmarins — APPROVED** [review]
nice job!

**Pull request merged**